Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayForPod.xcodeproj/xcshareddata/xcschemes/DaysUntilBirthday(iOS).xcscheme

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
For the two Pod project schemes, could you include a space between "DaysUntilBirthday" and "(iOS)" or "(macOS)"?  This would be consistent with the SPM project scheme names.